flloat/ltlf.py

```diff
@@ -544,6 +544,37 @@ def _delta(self, i: PropositionalInterpretation, epsilon=False):
         return self.to_nnf()._delta(i, epsilon=epsilon)
 
 
+class LTLfLast(LTLfFormula):
+    """Class for the LTLf Last formula."""
+
+    def _delta(self, i: PropositionalInterpretation, epsilon=False):
+        """Apply the delta function."""
+        return self.to_nnf()._delta(i, epsilon=epsilon)
+
+    def to_nnf(self) -> LTLfFormula:
+        """Transform to NNF."""
+        return LTLfAnd([LTLfWeakNext(LTLfFalse()), LTLfNot(LTLfEnd())]).to_nnf()
```

Comment:
I had to add the requirement `!End` because weak next is true on empty trace. So, `Last` would have been true even after the last symbol.

Reply:
I think that makes sense.